packages/trader/src/AppV2/Components/Guide/definition-modal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ActionSheet, Heading, Text } from '@deriv-com/quill-ui';
+import { Localize, localize } from '@deriv/translations';
+import { getTermDefinition } from 'AppV2/Utils/trade-types-utils';
+
+type TDefinition = { contract_type: string; term: string; onClose: () => void };
```

Comment:
```suggestion
type TDefinitionModal = { contract_type: string; term: string; onClose: () => void };
```
nit-pick ) or `TDefinitionModalProps`, up to you :)

Reply:
Sure, thank you)